juliac.jl

```diff
@@ -148,6 +153,10 @@ function julia_compile(julia_program, c_program=nothing, build_dir="builddir", v
     if object || shared || executable
         command = `"$(Base.julia_cmd())" "--startup-file=no" "--output-o" "$o_file" "-e"
                    "include(\"$julia_program\"); push!(Base.LOAD_CACHE_PATH, \"$julia_pkglibdir\"); empty!(Base.LOAD_CACHE_PATH)"`
+        # set compilation target
+        for i in eachindex(command.exec)
+            command.exec[i] = replace(command.exec[i], "x86-64", target)
```

Comment:
I would do this as `cmd = command.exec[i]; startswith(cmd, "-C") && command.exec[i] = "-C$target"` so that it won't accidentally corrupt the path to someone's julia executable or program.